components/PublicationTabs.tsx

```diff
@@ -29,6 +29,7 @@ export enum PublicationTab {
     PARTICIPANTS = 'participants',
     BIOSPECIMENS = 'biospecimens',
     TOOLS = 'tools',
+    SUPPORTING_LINKS = 'supporting_links',
```

Comment:
maybe good to use `supporting-links`? There are a few other tabs that have dashes instead of lowerdashes I believe